core/src/main/java/org/jdbi/v3/core/locator/internal/ClasspathBuilder.java

```diff
@@ -42,7 +42,7 @@ public ClasspathBuilder setExtension(@Nullable String extension) {
     }
 
     // org.foo.Bar$Inner -> org/foo/Bar$Inner
-    public ClasspathBuilder appendFullyQualifiedClassName(@Nonnull Class clazz) {
+    public ClasspathBuilder appendFullyQualifiedClassName(@Nonnull Class<?> clazz) {
```

Comment:
I think this change is ok, but it does change a signature (from raw type to any type). @stevenschlansker to double check? It is an internal class, though, so it should be fine.

Reply:
Of course we can remove this change if you feel unsure.

Reply:
Internal APIs are OK to change without regard for compatibility.